service/app/models/application_model.rb

```diff
@@ -0,0 +1,4 @@
+module ApplicationModel
+	extend ActiveSupport::Concern
```

Comment:
__[rubocop]__
* Use 2 (not 1) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Tab detected. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)